ThreadGroup/Controllers/ThreadNetworkConfig/TGNetworkConfigViewController.m

```diff
@@ -56,11 +56,16 @@ - (void)tableViewSetup {
 - (void)viewWillAppear:(BOOL)animated {
     [super viewWillAppear:animated];
     [self.navigationController setNavigationBarHidden:NO animated:YES];
-
+    self.navigationController.navigationBar.barTintColor = [UIColor threadGroup_orange];
     self.navigationController.navigationBar.backItem.title = @"";
     [self.navigationItem setTitle:@"Network Settings"];
 }
 
+- (void)viewWillDisappear:(BOOL)animated {
+    [super viewWillDisappear:animated];
+    self.navigationController.navigationBar.barTintColor = [UIColor threadGroup_grey];
```

Comment:
I think this could be moved into the viewWillAppear of TGNavigationController. So that if in the future we push more view controllers, or the navigation stack changes, we can be sure that TGMainVC and TGNoWifiVC will always have the right nav bar color